documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -259,6 +259,82 @@
               description: 'Prometheus %(prometheusName)s has missed {{ printf "%%.0f" $value }} rule group evaluations in the last 5m.' % $._config,
             },
           },
+
+          {
+            alert: 'PrometheusScrapeSeriesCardinality',
+            expr: |||
+              sum(scrape_series_added) by (job) > 100 * 1000
+            ||| % $._config,
+            'for': '30m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'Prometheus scrape series cardinality is exceeding server abilities.',
```

Comment:
If this is the case, the server is already dead. I don't think you'd be able to provide a generic value of use here.

In addition the scrape_ metrics are about the Prometheus you're talking to, not the one you're monitoring.